tests/conftest.py

```diff
@@ -334,6 +344,18 @@ def get_inputs(
 
         return all_inputs
 
+    def classify(self, prompts: List[str]) -> List[str]:
+        # output is final logits
+        all_inputs = self.get_inputs(prompts)
+        outputs = []
+        print(f"model: {self.model}")
```

Comment:
There are still a bunch of print statements in the code. Please remove them before finalizing this PR.

Reply:
I just pushed it.